backends/dpdk/CMakeLists.txt

```diff
@@ -122,6 +125,22 @@ set (P4_16_SUITES
   "${P4C_SOURCE_DIR}/testdata/p4_16_psa_errors/*.p4"
   "${P4C_SOURCE_DIR}/testdata/p4_16_dpdk_errors/*.p4"
   "${P4C_SOURCE_DIR}/testdata/p4_16_pna_errors/*.p4")
-p4c_add_tests("dpdk" ${DPDK_COMPILER_DRIVER} "${P4_16_SUITES}" "" "--bfrt")
+ p4c_add_tests("dpdk" ${DPDK_COMPILER_DRIVER} "${P4_16_SUITES}" "" "--bfrt") 
+
+# Add PTF tests for these files.
+set (DPDK_PTF_TEST_SUITES
+  "${P4C_SOURCE_DIR}/testdata/p4_16_samples/pna-dpdk-add_on_miss0.p4"
+)
+
+# This check should be improved
```

Comment:
CMake offers the option [find_program](https://cmake.org/cmake/help/latest/command/find_program.html) which lets you search for and initialize executables. The same thing is available for libraries. 

We can do this in the next PR.